src/CMakeLists.txt

```diff
@@ -69,6 +66,15 @@ if(FLB_STATS)
     )
 endif()
 
+if(FLB_BUFFERING)
```

Comment:
thanks for the review. Would you please if this is specificaly necessary ?, each buffering file associated have a conditional in the code

```
#ifdef FLB_HAVE_BUFFERING
...
#endif

the goal is to have the files there, if buffering is disabled that's fine because the pre-processor will make to skip the code.
```